src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -101,6 +111,12 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         _scrollBar.LargeChange(4);
         _scrollBar.Visibility(Visibility::Visible);
 
+        _tsfInputControl = TSFInputControl();
+        _tsfInputControl.CompositionCompleted({ this, &TermControl::_CompositionCompleted });
+        _tsfInputControl.CurrentCursorPosition({ this, &TermControl::_CurrentCursorPositionHandler });
+        _tsfInputControl.CurrentFontInfo({ this, &TermControl::_FontInfoHandler });
+        container.Children().Append(_tsfInputControl);

```

Comment:
we may need to tell the accessibility subsystem _something_ about this, but I am not sure what. @carlos-zamora?